2025/build-tooling/slides.md

```diff
@@ -19,7 +19,7 @@ is: feedback
 # Agenda
 
 - Introduction to build tools
-- Building an app (based on how Esri teams do it)
+- Building an app
```

Comment:
I think we can just speak to "Based on how Esri teams do it", and simplify this slide